src/xunit.analyzers/FixProviders/MemberFixBase.cs

```diff
@@ -27,7 +27,7 @@ public MemberFixBase(IEnumerable<string> diagnostics)
             if (declaringType == null)
                 return;
 
-            var member = declaringType.GetMembers(diagnostic.Properties["MemberName"]).FirstOrDefault();
+            var member = declaringType.GetMember(diagnostic.Properties["MemberName"]);
```

Comment:
Once again, not a bad change but I'd prefer to separate this sort of cleanup into different PRs/commits.

Reply:
OK, I'll open a separate PR for this. You mentioned earlier you were interested more in feature/bug PRs than refactoring PRs, so I was trying to sneak in a bit of refactoring into the former kind of PR.

Reply:
> trying to sneak in

being the operative word here :).

Please consider that I have a limited amount of personal time I'm willing to spend on this project and so the less "noise" I have to filter through, the more I can spend on the "signal".

Reply:
> > trying to sneak in
>
> being the operative expression here :).

I was half-joking there :smile: But anyway, here's what I was trying to get at in that comment: what kinds of refactoring PRs will you take in and what kinds will you reject? I'm just a bit confused because #72 was accepted and you comment here you'd like a different PR for the refactorings, but in https://github.com/xunit/xunit.analyzers/pull/73#issuecomment-322104265 you said

> I would much rather review and accept PRs that contain new features and bug fixes (as taken from the issue tracker) than refactorings and stylistic changes.

Reply:
I like PRs to contain a single commit that references an issue (bug or new feature) from the issue tracker. I use the commit history to collect release notes and so would rather have commits that express a self-contained piece of functional change.

I recognize that sometimes a functional change is easier to implement after refactoring the existing code. In those cases I prefer the PR to have one refactoring commit and one commit that implements the functional change. I will preserve the 2 commits via a rebase.

I rewrite local git history all the time and do not like seeing commits like "WIP", "fix the test", or "forgot to change the thing in this one place" within a PR. I see no value is such commits being preserved in the project's history and think they should be squashed whenever possible. I have no qualms about force pushing a branch.

For refactorings I like to see a meaningful improvement to the reusability/readability of the code. Factoring out identical or similar pieces of code into a reusable function is great. Stylistic (e.g. bracing style, whitespace) or inconsequential structural (e.g. expression bodied vs. traditional functions/properties, referencing different constants, changing a static to a const, extensions vs regular static calls) are not great. Drive-by tweaks to a file unrelated to the functional change addressed in the PR are simply an unnecessary distraction.

TL;DR: PRs that add value (new features/bug fixes) get accepted. PRs that move my cheese around get rejected.

PS: I'm a fickle and capricious person and can arbitrarily misapply these rules at any time.